tests/components/device_tracker/test_init.py

```diff
@@ -122,6 +122,7 @@ async def test_reading_yaml_config(
         0
     ]
     await hass.async_block_till_done()
+    await hass.async_block_till_done()
```

Comment:
The referenced test run didn't have any block till done.

Reply:
Already fixed with https://github.com/home-assistant/core/pull/113859